js/jquery.knob.js

```diff
@@ -779,9 +781,19 @@
                 c.stroke();
                 r = this.cv == this.v;
             }
+            var perc = (isNaN(this.v) ? 100 : this.v) / 100;
+            var gwidth = perc === 100 ? this.w : (this.w * perc);
+            var grad = c.createLinearGradient(0, this.h, gwidth, this.h);
+            grad.addColorStop(0, this.o.fgGradientStart);
+            grad.addColorStop(1, this.o.fgGradientEnd);
```

Comment:
I think that gwidth formula will be better like that (in case of negative this.o.min) :
var gwidth = (((this.cv - this.o.min) * this.w) / (this.o.max - this.o.min));

Using this.cv allows to change color dynamically.

The result is cool but why use gradients instead of changing the fgcolor of the entire circle while changing value ? with new props fgColorMin, fgColorMax for example ?